src/main/java/stringCalculator/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,40 @@
+package stringCalculator.calculator;
+
+import stringCalculator.common.Constant;
+import stringCalculator.number.Positive;
+import stringCalculator.utility.PatternChecker;
+import stringCalculator.utility.Separator;
+import stringCalculator.utility.Validator;
+
+public class StringAddCalculator {
+    Validator validator;
+    Separator separator;
+    Positive positive;
+
+    public StringAddCalculator() {
+        validator = new Validator();
+        separator = new Separator();
+        positive = new Positive(Constant.DEFAULT_POSITIVE_VALUE);
+    }
+
+    public int splitAndSum(String input) {
```

Comment:
```suggestion
    public int splitAndSum(String input) {
        if (!validator.isValidInput(input)) {
            return 0;
        }
        PatternChecker patternChecker = new PatternChecker(input);
        String text = patternChecker.getText();
        String delimiter = patternChecker.getDelimiter();
        return sumPositive(separator.splitNumber(text, delimiter));
    }
```

어떤 text와 delimiter를 반환해야할지 PatternChecker에서 결정해도 되지 않을까요?